staging/src/k8s.io/apiserver/pkg/storage/storagebackend/config.go

```diff
@@ -35,7 +35,8 @@ const (
 
 	DefaultCompactInterval      = 5 * time.Minute
 	DefaultDBMetricPollInterval = 30 * time.Second
-	DefaultHealthcheckTimeout   = 2 * time.Second
+	DefaultHealthcheckTimeout   = 15 * time.Second
```

Comment:
I understand the motivation and I'm supportive for the motivation as described in:
https://github.com/kubernetes/kubernetes/issues/111290

But technically that's a breaking change - for people who are not setting the timeout now, they will face a default behavior change.

For me it seems much safer to leave it set to 2s and just rely that people who would like to bump it will use the already existing flag. @deads2k - FYI

Reply:
I'm fine with leaving etcd as 2s and using existing flag to tune this.